pkg/datapath/linux/node_linux_test.go

```diff
@@ -1370,6 +1376,39 @@ func (s *linuxPrivilegedIPv4OnlyTestSuite) TestArpPingHandling(c *check.C) {
 		}
 	}
 	c.Assert(found, check.Equals, false)
+
+	c.Assert(linuxNodeHandler.NodeAdd(nodev3), check.IsNil)
+	time.Sleep(100 * time.Millisecond) // insertNeighbor is invoked async
```

Comment:
Please rebase against the latest master. https://github.com/cilium/cilium/pull/16072 removed the sleeps.

Reply:
@brb this is v1.8, not master